config/database.yml

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
replace with example config. Your personal local config should not be under version control.